package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperplay/ui",
-  "version": "1.18.3",
+  "version": "1.18.4",
```

Comment:
Just be aware that Flavio is about to merge a version with this same number :)

Reply:
REleased 0.19.0